### PR TITLE
Add PCT V space

### PR DESCRIPTION
**Pull request by percent-finance**

### CODE REVIEW

File: spaces/percent/skin.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.name {
+  --primary-color: #384aff;
+  --bg-color: white;
+  --text-color: #586069;
+  --link-color: #111111;
+  --heading-color: #111111;
+  --border-color: #d1d5da;
+  --header-bg: white;
+  --block-bg: transparent;
+}
```
Comment (e-asia) [inline spaces/percent/skin.scss:10]: To use the default theme you will want to delete skin.scss from your space folder.

**Comment by percent-finance** [comment]
@e-asia Thanks! The skin.scss is deleted now! Please check again!

File: spaces/percent/index.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  "name": "Percent",
+  "defaultView": "core",
+  "address": "0xbc16da9df0A22f01A16BC0620a27e7D6d6488550",
+  "token": "0xbc16da9df0A22f01A16BC0620a27e7D6d6488550",
+  "core": [
+    "0x4fB94CB25918Cbe62EC2ab80E3569492af59B8c3"
+  ],
+  "min": 0,
+  "invalid": []
+}
```
Comment (e-asia) [inline spaces/percent/index.json:13]: We have updated the format in `index.json`. 
Please add 2 new parameters `chainId` and `decimals`.
You can refer to the documentation that has been completed. https://docs.snapshot.page/guides/create-a-space#3-change-your-space-metadata

**Review by e-asia — CHANGES_REQUESTED** [review]

**Comment by percent-finance** [comment]
@e-asia Thanks! I've added the 2 new fields. I've also changed the symbol and address, so that it now supports quadratic voting. Please review again.

**Review by e-asia — APPROVED** [review]

**Pull request merged**